src/main/kotlin/me/zeroeightsix/kami/feature/module/RemoteCommand.kt

```diff
@@ -0,0 +1,74 @@
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
```

Comment:
```suggestion
package me.zeroeightsix.kami.feature.module

import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
```

Reply:
Modules must be in the module package to be eligible for the reflection scan.

Reply:
I didn't realize I somehow forgot to put it in the package... that was dumb